src/include/sof/audio/codec_adapter/codec/generic.h

```diff
@@ -57,7 +57,8 @@ struct codec_interface {
 	int (*process)(struct comp_dev *dev);
 	/**
 	 * Codec specific apply config procedure, called by codec_adapter every time
-	 * new configuration has been loaded.
+	 * a new RUNTIME configuration has been if in correct state. This will not
```

Comment:
The proposed sentence doesn't seem to be correct, what about
"...a new RUNTIME configuration has been sent, assuming codec is already prepared. This will not be used for SETUP config."

Reply:
yep, i think i missed a word, thanks